tests/ops/service/pagination/test_pagination_strategy_cursor.py

```diff
@@ -36,6 +36,36 @@ def response_with_empty_list():
     return response
 
 
+@pytest.fixture(scope="function")
+def response_with_has_next_true():
+    response = Response()
+    response._content = bytes(
+        json.dumps(
+            {
+                "conversations": [{"id": 1}, {"id": 2}, {"id": 3}],
+                "meta": {"next_cursor": 123, "has_next": "true"},
```

Comment:
Nice tests! 🙌🏽 

We could add the specific case when the values are not in the last "meta" object (AKA the one in my previous comment) for completion